transport/grpc/headers.go

```diff
@@ -213,7 +213,7 @@ func getApplicationHeaders(md metadata.MD) (transport.Headers, error) {
 		case 1:
 			value = values[0]
 		default:
-			return headers, yarpcerrors.InvalidArgumentErrorf("header has more than one value: %s", header)
+			return headers, yarpcerrors.InvalidArgumentErrorf("header has more than one value: %s:%v", header, values)
```

Comment:
Don't we need similar change for the respective test as well?

Reply:
follow up here: https://github.com/yarpc/yarpc-go/pull/2056